core/comments/rendered_workspace_comment.ts

```diff
@@ -178,11 +182,6 @@ export class RenderedWorkspaceComment
     }
   }
 
-  /** Returns whether this comment is deletable or not. */
-  isDeletable(): boolean {
-    return !this.workspace.options.readOnly;
-  }
-
```

Comment:
Was this removal intentional?

Reply:
Ah yes sorry, forgot to put a comment on this before opening the PR.

This was erroneously added. The base class has an implementation of this that doesn't need to be overriden.